int32 status
AntennaOffsetSet_t[] offsetSet
